lib/routers/tasks/index.js

```diff
@@ -8,15 +8,17 @@ const fetchTasks = () => {
       query: {
         data: {
           subject: req.user.profile.id
-        }
+        },
+        ...req.query
```

Comment:
what does req.query contain? what if the user decides to include a ?data=blah property?

Reply:
could we explicitly specify the properties to take from the query?